pkg/render/node.go

```diff
@@ -337,11 +337,19 @@ func (c *nodeComponent) nodeCNIConfigMap() *v1.ConfigMap {
 	}
 
 	// Determine MTU to use for veth interfaces.
-	var mtu int32 = 1410
+	// Zero means to use auto-detection.
+	var mtu int32 = 0
 	if m := getMTU(c.cr); m != nil {
 		mtu = *m
 	}
 
+	// Calico Enterprise doesn't do auto-MTU detection yet.
+	// If not specified, default it to the smallest value (wireguard) so that
+	// it will work in the most environments.
+	if c.cr.Spec.Variant == operatorv1.TigeraSecureEnterprise && mtu == 0 {
+		mtu = 1400
```

Comment:
The default for this was 1410 before. Should it have been 1400 and we were just lucky?

Reply:
Yeah, it should have been 1400 to account for wireguard potentially being enabled.